src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Empty div with class suggesting there should be a logo. Please check if nothing is missing. Add Logo image. Also, remember to add <a> tag to use it as a navigation element